src/test/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidatorTest.java

```diff
@@ -0,0 +1,100 @@
+package tds.exam.results.validation.impl;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import java.io.IOException;
+
+import tds.exam.results.configuration.ExamResultsTransmitterServiceProperties;
+import tds.exam.results.trt.TDSReport;
+import tds.exam.results.validation.TDSReportValidator;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+public class XSDBackedTDSReportValidatorTest {
+    private Resource resource;
+    private ExamResultsTransmitterServiceProperties properties;
+
+    @Before
+    public void setUp() throws Exception {
+        resource = new ClassPathResource("xsd/TestResultsTransmissionFormat_Schema.xsd");
+        properties = new ExamResultsTransmitterServiceProperties();
+        properties.setValidateTrtXml(true);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void shouldNotValidate() throws IOException, SAXException {
+        JAXBContext context = mock(JAXBContext.class);
+        properties.setValidateTrtXml(false);
+        TDSReportValidator validator = new XSDBackedTDSReportValidator(context, resource, properties);
+
+        TDSReport report = new TDSReport();
+
+        validator.validateReport(report);
+
+        verifyZeroInteractions(context);
+    }
+
+    @Test
+    @Ignore
```

Comment:
I ignored this for now because I time boxed this fix, and trying to figure out the proper formats for some of the String values wasn't clear to me.

@burzum619 - I haven't looked at the XSD, and it may be due to me misusing the API, but I hope that they provide enums, constants, or date setters for the date or enumerated value elements.  For example, `setCompleteness` takes in a String but there is a set of 3 valid String values that are allowed.